desktop/src/main/java/bisq/desktop/main/MainView.java

```diff
@@ -429,6 +431,18 @@ public void onCheckpointFail() {
                 .useShutDownButton()
                 .show();
     }
+    
+    private void checkOpenSupportTickets() {
+		// Create modal pop-up with single button taking user to Support screen if there
+		// are active support tickets
+		if (model.getNumOpenSupportTickets() != null && Integer.parseInt(model.getNumOpenSupportTickets().get()) > 0) {
+			new Popup<>().attention(Res.get("portfolio.pending.support.popup.attention"))
+					.closeButtonTextWithGoTo("navigation.support")
+					.onClose(
+							() -> navigation.navigateTo(MainView.class, SupportView.class, ArbitrationClientView.class))
+					.show();
```

Comment:
I think we should add a don't show again option for power users that might get annoyed having it popup every time it opens as long they have a support case open.

Reply:
I will look into that. Thanks.

Reply:
I assume by power user, you refer to an Arbitrator/DisputeAgent?

Reply:
This PR has moved to:

https://github.com/bisq-network/bisq/pull/3477/commits/aae6fc7b6fd07152d1b475593dd9f14b96e6d037